DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -33,6 +33,9 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
             // percentage completion set to 0.
             // This is achieved using LEFT JOINs on Progress, so we get the candidates progress details or some nulls.
 
+            // This query starts by getting a record per tutorial (with progress for that tutorial from aspProgress)
+            // and then aggregates them into a list of sections with tutorial percentage completion of each section.
+
             CourseContent? courseContent = null;
             return connection.Query<CourseContent, CourseSection, CourseContent>(
                 @"  WITH CustomisationDurations AS (

```

Comment:
Would probably be useful with a comment explainin what the CustomisationDurations is for and why we need a separate view for it 🙂

Reply:
Yep! Added in c6e65d9 🙂